src/import/engine.rs

```diff
@@ -254,6 +256,13 @@ impl SSTWriter {
         })
     }
 
+    fn tune_config(env: Arc<Env>, cf_opts: &mut ColumnFamilyOptions) {
+        use DBCompressionType::Lz4;
+        cf_opts.set_env(env);
+        cf_opts.compression_per_level(&[Lz4, Lz4, Lz4, Lz4, Lz4, Lz4, Lz4]);
```

Comment:
IMO, compression can increase CPU usage.

Reply:
The SstFileWriter only uses the [compression of the last level](https://github.com/facebook/rocksdb/blob/18864567c8a1139d2fc83cda24587043d5beaf84/table/sst_file_writer.cc#L197-L200) (Level 6). The default setting uses `Zstd` which [has higher compression ratio but is slower](https://stackoverflow.com/a/40151577). 

For sure we could set it to `&[No, No, No, No, No, No, Lz4]` but the outcome is the same.

Reply:
@kennytm addressed